helm/values.yaml

```diff
@@ -107,6 +107,7 @@ rawSpansGrouperConfig:
         numStreamThreads: 4
         bootstrapServers: "bootstrap:9092"
         schemaRegistryUrl: "http://schema-registry-service:8081"
+        cacheMaxBytesBuffering: "104857600"
```

Comment:
This value seems a bit arbitrary, do you mind elaborating on why exactly this value? Also I wonder if this should be a string instead of an int.